metadata.json

```diff
@@ -73,6 +73,12 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Gentoo",
+      "operatingsystemrelease": [
+        "1.0"
```

Comment:
To be honest, i'm not quite sure what to put there. Gentoo doesn't have a real "operatingsystemrelease" variable. It uses the kernel release value. I've just copied this from puppetlabs-firewall as this is a official module which has Gentoo support.